src/helm/benchmark/run_specs.py

```diff
@@ -2243,6 +2285,198 @@ def get_anthropic_hh_rlhf_spec(num_respondents: int, subset: str) -> RunSpec:
     )
 
 
+@run_spec_function("stereotype_bias")
```

Comment:
For these, could you prefix the name with "decodingtrust" e.g. "decodingtrust_stereotype_bias"? Most of these names are very generic e.g. "fairness", so it could be easily confused with other scenarios.

Reply:
Thanks for the suggestion! I modified `run_spec.py` and `run_specs_decodingtrust.conf` with the modified scenario names for for clarity.